frontend/src/patientApp/timeOfTest/TestResult.test.tsx

```diff
@@ -58,23 +51,30 @@ describe("TestResult", () => {
     );
 
     expect(screen.getByText("SARS-CoV-2 result")).toBeInTheDocument();
+    expect(screen.getByText("Abbott BinaxNOW (Antigen)")).toBeInTheDocument();
     expect(screen.getByText("Bob Barker")).toBeInTheDocument();
     expect(screen.getByText("Test result")).toBeInTheDocument();
     expect(screen.getByText("Negative")).toBeInTheDocument();
   });
   it("should show an inconclusive result", () => {
-    const store = mockStore({
-      plid: "foo",
-      patient: {
-        firstName: "Bob",
-        lastName: "Barker",
-        lastTest: {
-          deviceType: "Testing device",
-          dateTested: "08/27/2020",
-          result: "UNDETERMINED",
-        },
-      },
-    });
+    const store = mockStore(getPatientLinkData("UNDETERMINED"));
+    render(
+      <Router history={createMemoryHistory()}>
+        <Provider store={store}>
+          <TestResult />
+        </Provider>
+      </Router>
+    );
+
+    expect(screen.getByText("SARS-CoV-2 result")).toBeInTheDocument();
+    expect(screen.getByText("Abbott BinaxNOW (Antigen)")).toBeInTheDocument();
+    expect(screen.getByText("Bob Barker")).toBeInTheDocument();
+    expect(screen.getByText("Test result")).toBeInTheDocument();
+    expect(screen.getByText("Inconclusive")).toBeInTheDocument();
+  });
+
+  it("should show the patient/device name", () => {
```

Comment:
I think this is an exact copy of the above test now, it may be safe to remove.

Reply:
good catch, will restructure these tests to be more atomic